tensorbay/cli/branch.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+"""Implementation of gas branch."""
+
+import sys
+from typing import Dict
+
+import click
+
+from ..client.gas import DatasetClientType
+from ..client.requests import PagingList
+from ..client.struct import Branch
+from .tbrn import TBRN, TBRNType
+from .utility import get_dataset_client, get_gas
+
+DEFAULT_BRANCH_NAME = "main"
```

Comment:
the default branch can be changed,
make OpenAPI tell you what branch is default branch.